README.md

```diff
@@ -1,3200 +1,18 @@
- * @return The type of the underlying object for this binary.
- */
-serialization::pimpl::object_type get_type() const;
-
-/**
- * Deserializes the underlying binary data and produces the object of type T.
- *
- * <b>CAUTION</b>: The type that you provide should be compatible with what object type is returned with
- * the get_type API, otherwise you will either get an exception of incorrectly try deserialize the binary data.
- *
- * @tparam T The type to be used for deserialization
- * @return The object instance of type T.
- */
-template <typename T>
-boost::optional<T> get() const;
-```
-
-typed_data does late deserialization of the data only when the get method is called.
-
-This typed_data allows you to retrieve the data type of the underlying binary to be used when being deserialized. This class represents the type of a Hazelcast serializable object. The fields can take the following values:
-1. <b>Primitive types</b>: `factory_id=-1`, `class_id=-1`, `type_id` is the type ID for that primitive as listed in `serialization_constants`
-2. <b>Array of primitives</b>: `factory_id=-1`, `class_id=-1`, `type_id` is the type ID for that array as listed in `serialization_constants`
-3. <b>identified_data_serializer serialized objects</b>: `factory_id`, `class_id` and `type_id` are non-negative values.
-4. <b>Portable serialized objects</b>: `factory_id`, `class_id` and `type_id` are non-negative values.
-5. <b>Custom serialized objects</b>: `factory_id=-1`, `class_id=-1`, `type_id` is the non-negative type ID. 
-
-# 8. Development and Testing
-
-Hazelcast C++ client is developed using C++. If you want to help with bug fixes, develop new features or
-tweak the implementation to your application's needs, you can follow the steps in this section.
-
-## 8.1. Testing
-
-In order to test Hazelcast C++ client locally, you will need the following:
-
-* Java 8 or newer (for server)
-* Maven
-* cmake
-* OpenSSL
-* Boost
-* thrift
-
-You need to enable the examples in the build with the cmake flag `-DBUILD_EXAMPLES=ON`. When you build, the result will
-produce the executable `client_test` on Linux/macOS and `client_test.exe` on Windows. You need to start the remote
-controller first before running this executable by executing `scripts/start-rc.sh` on Linux/macOS
-or `scripts/start-rc.bat` on Windows. Then you can run the test executable.
-
-# 9. Getting Help
-
-You can use the following channels for your questions and development/usage issues:
-
-* This repository by opening an issue.
-* Hazelcast Hazelcast channel on Gitter: 
-[![Join the chat at https://gitter.im/hazelcast/hazelcast](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/hazelcast/hazelcast?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-* Our Google Groups directory: https://groups.google.com/forum/#!forum/hazelcast
+* [Hazelcast Community Slack](https://slack.hazelcast.com/)
+* Hazelcast Hazelcast channel on [Gitter](https://gitter.im/hazelcast/hazelcast)
```

Comment:
I don't know if it is intentional, but still: "Hazelcast" is duplicated here.

Reply:
fixed.